dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/testing/ContributionIntegrationTest.java

```diff
@@ -88,29 +88,29 @@ public void setupHelper(){
     @Test
     void testProcessContributionUpdateWhenNotFound() {
         final String errorText = "The request has failed to process";
-        final var updateLogContributionRequest = UpdateLogContributionRequest.builder()
+        final var updateLogContributionRequest = ContributionProcessedRequest.builder()
                 .concorId(9)
                 .errorText(errorText)
                 .build();
-        softly.assertThatThrownBy(() -> contributionService.processContributionUpdate(updateLogContributionRequest))
+        softly.assertThatThrownBy(() -> contributionService.handleContributionProcessedAck(updateLogContributionRequest))
                 .isInstanceOf(WebClientResponseException.class);
         assertProcessConcorCaseSubmissionCreation(updateLogContributionRequest, HttpStatus.NOT_FOUND);
     }
 
     @Test
     void testProcessContributionUpdateWhenFound() {
         final String errorText = "Error Text updated successfully.";
-        final var updateLogContributionRequest = UpdateLogContributionRequest.builder()
+        final var updateLogContributionRequest = ContributionProcessedRequest.builder()
```

Comment:
Same as above: This local variable name seems to be a hangover from the previous name and can be changed to contributionProcessedRequest?

Reply:
Yep, both fixed. Also fixed the same issue in the FDC test.